graphdatascience/graph/graph_entity_ops_runner.py

```diff
@@ -164,7 +166,9 @@ def _process_result(
             unique_node_ids = result["nodeId"].drop_duplicates().tolist()
 
             db_properties_df = query_runner.run_retryable_cypher(
-                GraphNodePropertiesRunner._build_query(db_node_properties), params={"ids": unique_node_ids}
+                GraphNodePropertiesRunner._build_query(db_node_properties),
+                params={"ids": unique_node_ids},
+                mode=QueryMode.READ,
```

Comment:
This is a hard-coded change. We will now run with `READ` for a number of operations, instead of `WRITE`. I think we should call that out in the changelog as well.